### ISSUE TITLE
Cannot read properties of undefined (reading 'length')

### ISSUE THREAD

**Issue by ronenteva** [mvrlin/nuxt-viewport#29]
I can't replicate the issue in dev, but on the server I keep getting the error referring to `parseCookie`:

```
export function parseCookie(input: string): Record<string, string> {
  if (!input.length) {
    return {}
  }

  return Object.fromEntries(input.split(/; */).map((cookie) => cookie.split('=', 2)))
}
```

**Comment by mvrlin** [comment]
@ronenteva what version of Nuxt are you using?

**Comment by ronenteva** [comment]
The latest - `3.0.0`

**Comment by mvrlin** [comment]
@ronenteva should be fixed in the latest release 2.0.3
